coderefinery-mooc.rst

```diff
@@ -0,0 +1,186 @@
+CodeRefinery MOOC strategy
+==========================
+
+This page documents the CodeRefinery MOOC strategy.  It is not a real
+MOOC (it's not massive enough yet), but it does reach out from one to
+many, and can scale to basically all the world.
+
+
+Technical setup summary
+-----------------------
+
+We have a public broadcast, with goes out via a livestream.
+Disconnected from this, people are watching the broadcast in a
+*separate* Zoom meeting and doing exercises/breakouts there.  Or,
+people can watch via the livestream alone.  Or there can be different
+meetings.  Or people could watch recorded videos later.
+
+The mental model here is "Watching TV together".  We collective watch
+a show together.  There are periodic intermissions where each watching
+community discusses among themselves and works on the exercises.
+Everyone feels they are a part of something big and that keeps people interested.
+
+We have clear communication channels from learner→instructor
+(HackMD), helpers→instructors (chat), instructors→learners
+(livestream).  Of course instructors can directly to the audience
+during their breaks.
+
+The **Director** controls the stream and is responsible for keeping
+things running smoothly.
+
+
+
+Instructors
+-----------
+
+There is an **instructor Zoom meeting**.  This is broadcasted via
+Twitch, using :doc:`OBS <obs>` (there is usually a separate director
+or production manager for this, instructors don't need to worry
+themselves with this).
+
+Compared to the classic style, advantages include:
+
+* You are freed from student management, others help manage the
+  audience and convey these important parts to you.
+* Audio/video is muted during breaks, there is more opportunity to
+  discuss and plan what comes next with the instructors.
+
+Disadvantages:
+
+* You lose the direct access to all students (but how often would
+  someone speak up anyway?).
+
+
+Instructors should keep in mind:
+
+
+* At all times you will have a director to help keep you on track:
+  just teach and watch chat (and HackMD when you have time, but others
+  do this and let you know).
+* You will have a private Zoom meeting with only instructors (and any
+  other key helpers who want to be there).
+* Share a vertical screen (840 × 1080 maximum).  This allows students
+  to keep half of their screen open for their own work.
+* HackMD is the main way of receiving questions from students (just
+  like in our current courses).
+* During breaks and pauses, the livestream will be muted, so that
+  instructors (and helpers there) can talk without the audience
+  hearing.  This greatly increases professionalism and makes it easier
+  to coordinate.
+* There is the standard text chat (Zulip) to use to communicate with
+  other helpers.
+* Of course, you can go join the student room during breaks, other
+  sessions, and so on.
+
+* Zoom polls won't work.  We can use another service for that instead
+  (e.g. presemo.aalto.fi).
```

Comment:
```suggestion
  (e.g. presemo.aalto.fi). But also HackMD can be used for polls.
```

Reply:
Rephrased differently in follow-up to make HackMD the main emphasis.